alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -143,6 +145,66 @@ private function appendUiConfToContent($content)
 
 		return $content;
 	}
+
+	private function appendEnvConfigToContent($content)
+	{
+
+		$protocol = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] == 'on') ? "https" : "http";
+
+		// The default Kaltura service url:
+		$serviceUrl = requestUtils::getApiCdnHost().'/api_v3';
+		// Default Kaltura CDN url:
+		$cdnUrl = requestUtils::getCdnHost($protocol);
```

Comment:
@yossipapi do you know why `requestUtils::getCdnHost` requires passing protocol while in 
 `requestUtils::getApiCdnHost` the protocol is resolved inside the utils method?

Reply:
No obvious reason.